echopype/echodata/convention/1.0.yml

```diff
@@ -12,43 +12,84 @@ groups:
     name: Top-level
     description: contains metadata about the SONAR-netCDF4 file format.
     ep_group:
+    coords_ek60: []
+    coords_ek80: []
+    coords_azfp: []
+    coords_ad2cp: []
```

Comment:
For the top-level (root of file) group, I don't think we need to have coordinates, since it is a collection of global attributes.

Reply:
That makes sense, I will remove that.

Reply:
unless we produce a coordinate to list all the files that got combined together into a single file (using `combine_echodata`)  -- I haven't looked into the code to see if we ended up doing it, the memory is fading on me

Reply:
It looks like this is put in the group `provenance`. See below

https://github.com/OSOceanAcoustics/echopype/blob/1afc16aeb9b7ed67d39006ac9192f13b9d76e4b6/echopype/echodata/combine.py#L293-L299

Reply:
Ah nice discovery! I had completely forgotten about it...